src/edu/ucsb/nceas/metacat/restservice/v1/CNResourceHandler.java

```diff
@@ -520,7 +524,7 @@ private void getLog() throws InvalidToken, ServiceFailure, NotAuthorized,
         }
 
         logMetacat.debug("calling getLogRecords");
-        Log log = CNodeService.getInstance(request).getLogRecords(session,
+        org.dataone.service.types.v1.Log log = CNodeService.getInstance(request).getLogRecords(session,
```

Comment:
I added a new imported  class `org.apache.commons.logging.Log` since each ResourceHandler class has its own log object. So there are duplicate Log class and I have to use the full name here.